migrations/applied_migrations/Deregistered.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
do we want to delete these now or just all at the end?

Reply:
Can delete if we want to tidy up.